packages/react-components/react-shared-contexts/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -0,0 +1,184 @@
+/* eslint-disable */
+
+import * as React from 'react';
+
+// The list of hooks is built by scraping the exports from react-components/src/index
```

Comment:
Do we plan to introduce validation that we have full coverage of these style hooks here?

The script used to generate this that you used being used as a commmand would also be nice

Reply:
There is no script.  I scraped by hand.  I can open a work item to figure out how to lint or validate this.

Reply:
Please do, we should not be manually checking this

Reply:
+1 to Ling's comment. I think previous versions of Fluent had conformance tests for stuff like this. We should also figure out if we only want to check this for stable like what you have here @GeoffCoxMSFT and if there is a way to validate unstable components to check them before we move them from unstable to stable.